src/actors/common/cache.rs

```diff
@@ -285,13 +285,12 @@ impl<T: CacheItemRequest> Cacher<T> {
             // get a channel that will receive data, or they create a new channel.
             drop(remove_computation_token);
             sender.send(result).ok();
-            Ok(())
         }
         .bind_hub(Hub::new_from_top(Hub::current()));
 
         // TODO: This spawns into the arbiter of the caller. Consider more explicit resource
```

Comment:
maybe update the comment?

Reply:
The comment still holds up, I'll replace "arbiter" with "single-threaded runtime" though.